local-testing-docs/README.md

```diff
@@ -857,5 +857,10 @@ Please note that there will be unaware limitations, in this case please feel fre
   - Results of calculation regarding the values and data types
   - Columns names
   - Error experience
+- For data types Variant, Array and Object, basic capabilities of `parse_json`,
+  `to_array`, `to_object` and `to_variant` are provided via Python built-in JSONEncoder and
+  JSONDecoder which only support conversion of primitive Python data types. Advanced data conversion can be achieved by customizing
+  the `snowflake.snowpark.mock.connection._CUSTOM_JSON_ENCODER` and `snowflake.snowpark.mock.functions._CUSTOM_JSON_DECODER`.
+- Coercion is not supported
```

Comment:
Implicit type casting (Coercion is not supported). In general, please avoid using passing in arguments of different types to functions.

Reply:
updated